src/pkg/inspection/controller.go

```diff
@@ -139,6 +143,10 @@ func (c *controller) Run(ctx context.Context, policy *v1alpha1.InspectionPolicy)
 	// Just in case.
 	if len(nsl) == 0 {
 		log.Info("no namespaces found")
+		err2 := c.checkAndSendReportToGovernor(ctx, policy, &v1alpha1.AssessmentReport{})
+		if err2 != nil {
```

Comment:
Why do we send an empty report to the governor when no namespace label selector matches?

Reply:
**Scenario is**: Suppose user has given label to one namespace and after sometime, if user has removed that label. Next time, agent will not found any namespace to scan. But, governor will be still showing the telemetry data to user. So, to remove discrepancy in data from both side, we did.